postgresql.init.in

```diff
@@ -4,13 +4,13 @@
 #
 # This is the init script for starting up the PostgreSQL server.
 #
-# This script is slightly unusual in that the name of the daemon (postmaster)
+# This script is slightly unusual in that the name of the daemon (postgres)
 # is not the same as the name of the subsystem (postgresql)
 #
 # chkconfig: - 64 36
 # description: PostgreSQL database server.
-# processname: postmaster
-# pidfile: /var/run/postmaster.PORT.pid
+# processname: postgres
+# pidfile: /var/run/postgres.PORT.pid
```

Comment:
I'd drop this instead of changing;  the file doesn't exist on this location nowadays.

Reply:
+1